lib/ts/Vec.h

```diff
@@ -853,9 +853,8 @@ template <class C, class A, int S>
 inline void
 Vec<C, A, S>::free_and_clear()
 {
-  for (int x = 0; x < n; x++)
-    if (v[x])
-      A::free(v[x]);
+  for (size_t x = 0; x < (n); x++)
```

Comment:
This is going to be problematic since `Vec.h` has been removed. You will likely need to convert use of `Vec` to `std::vector`.

Reply:
yah..  :-<